src/Vimeo/Vimeo.php

```diff
@@ -31,8 +31,8 @@ class Vimeo
     const ACCESS_TOKEN_ENDPOINT = '/oauth/access_token';
     const CLIENT_CREDENTIALS_TOKEN_ENDPOINT = '/oauth/authorize/client';
     const REPLACE_ENDPOINT = '/files';
-    const VERSION_STRING = 'application/vnd.vimeo.*+json; version=3.2';
-    const USER_AGENT = 'vimeo.php 1.3.0; (http://developer.vimeo.com/api/docs)';
+    const VERSION_STRING = 'application/vnd.vimeo.*+json; version=3.4';
```

Comment:
tus works for versions prior to 3.4. Should we support that here or nah?

Reply:
Nah, we're making 3.4 the new default API version.